basic/x16additions.s

```diff
@@ -624,15 +633,10 @@ setbank:
 ;to the BASIC editor to prevent 40/80 key
 ;presses during program execution to take effect
 clear_4080_flag:
-	lda ram_bank
-	pha
-	lda #BANK_KERNAL
-	sta ram_bank
+	stz ram_bank
```

Comment:
FYI, #BANK_KERNAL is a symbol for the kernal ROM bank, not the RAM bank, so even though it's also zero, it's not quite the right constant.

BASIC is an end user of the `ram_bank` (ZP `$00`) setting.  I don't think there's any need for BASIC to jump through hoops to preserve the old value, since outside of interrupt handlers, BASIC is the sole consumer of banked RAM while it is running.